settings.gradle

```diff
@@ -107,3 +107,6 @@ include 'release:maven'
 include 'e2e-test:peerforwarder'
 include 'rss-source'
 include 'data-prepper-plugins:failures-common'
+include 'data-prepper-plugins:parquet-codecs'
+include 'data-prepper-plugins:newline-codecs'
+include 'data-prepper-plugins:avro-codecs'
```

Comment:
This is now in a different PR. Please remove this line so that the build passes.

Reply:
done